activate

```diff
@@ -72,15 +76,16 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 live=false
 
 # report in
-if ! out=$(wget -O - -q "$SCRIPT?image=$image&release=$release&vendor=$vendor&product=$product&serial=$serial&live=$live"); then
-    echo "wget failed"
+URL="${SCRIPT}?image=${image}&release=${release}&vendor=${vendor}&product=${product}&serial=${serial}&live=${live}"
+if ! out=$(wget -O - -q ${URL}); then
```

Comment:
this does what we expect - assignments do not reset the exit state. I've separated running the command from testing the exit status.